push.py

```diff
@@ -145,7 +145,7 @@ def generate_commit_message(output_dir, success):
 
     ccall("git clone https://github.com/precice/precice_st_output")
 
-    test_type = "Test" if args.test == "bindings" else "TestCompose"
+    test_type = "Test" if args.test in ["bindings","1dtube_py","1dtube_cxx"] else "TestCompose"
```

Comment:
Due to the simpicity of the tutorial I chose to create a standard Dockerfile instead of using Docker-compose.